bazel/sh_run/exec.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Don't pass BUILD_WORKING_DIRECTORY to the subcommand because args will use
+# pwd-relative paths.
+unset BUILD_WORKING_DIRECTORY
```

Comment:
The `$(location %s)` is pwd-relative, but there could be other args passed on the `blaze run` command line that aren't, such as `--trace_file`. I wonder if instead of doing this it would be preferable to form an absolute path in `bazel/sh_run/rules.bzl`?

I think this is minor though, and if it's not easy then we shouldn't worry about it and should land this as-is.

Reply:
WDYT of this realpath approach?

It can't be done from within rules.bzl; it's trying to force builds to remain hermetic wrt build path.